README.rst

```diff
@@ -1,23 +1,150 @@
+  --certificates-collection CERTIFICATES_COLLECTION
+                        Collection name for certificates
+  --gfx-bucket GFX_BUCKET
+                        Bucket name for gfx
+  --gfx-collection GFX_COLLECTION
+                        Collection name for gfx
+  --addons-bucket ADDONS_BUCKET
+                        Bucket name for addons
+  --addons-collection ADDONS_COLLECTION
+                        Collection name for addon
+  --plugins-bucket PLUGINS_BUCKET
+                        Bucket name for plugins
+  --plugins-collection PLUGINS_COLLECTION
+                        Collection name for plugin
+  -C, --certificates    Only import certificates
+  -G, --gfx             Only import GFX drivers
+  -A, --addons          Only import addons
+  -P, --plugins         Only import plugins
+
+
+
+Generate a blocklist.xml file from Kinto collections
+====================================================
+
+If you want to export blocklists stored in Kinto in the AMO XML export
+format, you can use the ``kinto2xml`` script::
+
+    kinto2xml -s http://localhost:8888
+
+
+**kinto2xml** gives you the ability to configure what you want to
+export and in which bucket and collection are the data stored.
+
+.. code-block::
+
+  -h, --help            show this help message and exit
+  -s SERVER, --server SERVER
+                        The location of the remote server (with prefix)
+  -a AUTH, --auth AUTH  BasicAuth token:my-secret
+  -v, --verbose         Show all messages.
+  -q, --quiet           Show only critical errors.
+  -D, --debug           Show all messages, including debug messages.
+  --certificates-bucket CERTIFICATES_BUCKET
+                        Bucket name for certificates
+  --certificates-collection CERTIFICATES_COLLECTION
+                        Collection name for certificates
+  --gfx-bucket GFX_BUCKET
+                        Bucket name for gfx
+  --gfx-collection GFX_COLLECTION
+                        Collection name for gfx
+  --addons-bucket ADDONS_BUCKET
+                        Bucket name for addons
+  --addons-collection ADDONS_COLLECTION
+                        Collection name for addon
+  --plugins-bucket PLUGINS_BUCKET
+                        Bucket name for plugins
+  --plugins-collection PLUGINS_COLLECTION
+                        Collection name for plugin
+  --app APP             Targeted blocklists.xml APP id
+  -o OUT, --out OUT     Output XML file.
+
+
+Show differences between two XML files
+======================================
+
+Once you've imported the data and exported them, you may want to
+validate that both Kinto and AMO export the exact same data in XML.
+
+You can use the ``xml-verifier`` command to validate that both files
+are rendered the same way::
+
+    xml-verifier \
+        https://blocklist.addons.mozilla.org/blocklist/3/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/46.0/ \
+        https://kinto-reader.dev.mozaws.net/v1/blocklist/3/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/46.0/
```

Comment:
What is this URL ? 
I suggest that you use the local files first, and then mention that it supports a remote URL of a Kinto server with the kinto-amo plugin install.

Would be nice to give the equivalent kinto2xml command along: 

```
APPID="{ec8030f7-c20a-464f-9b0e-13a3a9e97384}"
kinto2xml --app $APPID -o generated-blocklists.xml
xml-verifier https://blocklist.addons.mozilla.org/blocklist/3/$APPID/46.0/ generated-blocklists.xml
```